static/panes/compiler.js

```diff
@@ -1184,6 +1184,10 @@ Compiler.prototype.setAssembly = function (result, filteredCount) {
     this.assembly = asm;
     if (!this.outputEditor || !this.outputEditor.getModel()) return;
     var editorModel = this.outputEditor.getModel();
+    if (result.languageId)
+        monaco.editor.setModelLanguage(editorModel, result.languageId)
+    else
```

Comment:
It might be an issue for the language in the switch in `static/panes/compiler.js:134`